app/models/incident.rb

```diff
@@ -224,6 +224,38 @@ def to_csv
     end
   end
 
+  def self.to_csv
+    CSV.generate do |csv|
```

Comment:
For the most part copied over from the `to_csv` instance method.

Reply:
Seems like an ideal place for some abstraction, then:

```ruby
def self.to_csv
  CSV.generate do |csv|
    all.each do |incident|
      csv << incident.csv_row
    end
  end
end

def to_csv
  CSV.generate do |csv|
      csv << csv_row
  end
end

private

def csv_row
  row = []
  # shovel_party
  row
end

Reply:
AH thanks @werebus, exactly what I needed!